keyboards/lets_split/keymaps/geripgeri/keymap.c

```diff
@@ -0,0 +1,438 @@
+#include QMK_KEYBOARD_H
+
+enum layer_names {
+   _QWERTY,
+   _LOWER,
+   _RAISE,
+   _NUMPAD,
+   _GAMING,
+   _ADJUST
+};
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
```

Comment:
If you're only using the one default layer, you can safely remove this, actually.